ci/ci/github.py

```diff
@@ -51,6 +51,20 @@ class GithubStatus(Enum):
     FAILURE = 'failure'
 
 
+def github_status(state: str) -> GithubStatus:
+    """
+    Converts a state for a commit status or a conclusion for a check from the GraphQL API to a GithubStatus.
```

Comment:
Could you possibly document where the status/conclusion enums are defined in the github docs?

Reply:
added!